segmentation/utils/synthetic_spatial_datagen_test.py

```diff
@@ -10,7 +10,7 @@
 importlib.reload(synthetic_spatial_datagen)
 
 
-def test_direct_init_dist_matrix():
+def test_generate_test_dist_matrix():
```

Comment:
Can you add a simple test for the other two functions similar to what's here?
Doesn't have to be fancy, even a single assert is fine. Mostly just to catch any potential syntax errors/input dimension errors that could get accidentally introduced otherwise.

Reply:
Ok, got it.